sonar-java-plugin/src/test/java/org/sonar/plugins/java/SanityTest.java

```diff
@@ -109,7 +110,7 @@ class SanityTest {
    * It does not prevent other rules to fail if similar construct of the language, but not yet encountered.
    */
   @Test
-  @EnabledIfSystemProperty(named = "force.sanity.test", matches = "true")
+  //@EnabledIfSystemProperty(named = "force.sanity.test", matches = "true")
```

Comment:
Is this commented code intentional? Better to revert or remove in case it is not needed